.github/release-drafter.yml

```diff
@@ -3,14 +3,14 @@ tag-template: 'v$NEXT_PATCH_VERSION'
 categories:
   - title: 'New Features'
     label: 'feature'
-  - title: 'Breaking'
-    label: 'breaking'
+  - title: 'Breaking Change'
+    label: 'breaking change'
   - title: 'Bug Fixes'
     label: 'fix'
   - title: 'Documentation'
     label: 'documentation'
-  - title: 'Maintenance'
-    label: 'maintenance'
```

Comment:
maintenance means things like fixing the CI, adding linters, etc. Should we keep it as well? It's not clear to me what dependencies mean